README.md

```diff
@@ -462,6 +462,8 @@ Reference:
 By default, Process Migration service generates an in-memory database schema automatically.
 You should be able to disable it and manage the schema creation by setting and passing the `quarkus.flyway.migrate-at-start=false` system property.
 
+**Note**: Schema auto-creation for other databases different from H2 in-memory database is not supported due to some Flyway constraints and limitations.
```

Comment:
Can you add more details about where to find the ddl-scripts for all the supported databases?

Reply:
done